cmd/zc_traverser_blob.go

```diff
@@ -171,6 +171,15 @@ func (t *blobTraverser) traverse(preprocessor objectMorpher, processor objectPro
 				return fmt.Errorf("cannot list files due to reason %s", err)
 			}
 
+			// queue up the sub virtual directories if recursive is true
+			if t.recursive {
+				for _, virtualDir := range lResp.Segment.BlobPrefixes {
+					enqueueDir(virtualDir.Name)
+				}
+			}
+
+			marker = lResp.NextMarker
```

Comment:
Minor: did you intend to move this to the middle of the loop?  It looks unusual here.  Might be nice for maintainers if it was at the end like in all our other marker-based loops.